packages/vue/src/components/utils/agenda-list-item.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="agenda-list-item">
+    <div class="time">
+      <div class="start">
+        {{ startTime }}
+      </div>
+      <div class="duration">
+        {{ duration }}
+      </div>
+    </div>
+    <div class="description">
+      <div class="description-title">
+        {{ appointment.text }}
+      </div>
+      <div class="description-resource">
+        {{ resources[appointment.calendarId]?.text }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+import { Duration } from 'luxon';
+
+const props = withDefaults(defineProps<{
+  appointment: {startDate: Date},
+  resources: Record<string, unknown>[],
+}>(), {
+  items: () => ({}),
+  resources: () => [],
+});
+
+const startTime = computed(() => props.appointment.startDate.toLocaleTimeString(undefined, {
+  hour: 'numeric',
+  minute: 'numeric',
+  hour12: false,
```

Comment:
The scheduler shows a 12-hour time, while the agenda view and tooltip show a 24-hour time.

Let's be consistent, choose a format and use it everywhere?

Reply:
looks like it need to discuses for next step improvements with designer